### PR TITLE
 3 files added

### PR DESCRIPTION
**Pull request by Saqibi4213**
This is Portfolio about ME ! 
i worked on it with patient and alot of eforts 
trying to make portfolio professional and 
i worked on it with two code languages 
HTML , CSS 
linters also added 
and deployed link 
added live link also 
mentioned my partner to review my repo and project

### CODE REVIEW

**Comment by Saqibi4213** [comment]
hello dear @fatiabdulahi please checkout if their is any thing need to add or not orr any error

**Comment by Saqibi4213** [comment]
hello dear @fatiabdulahi  please review my pull request

**Comment by ahzamir** [comment]
@fatiabdulahi

- Although it's been more than 24 hours that your partner asked you for a review and feedback you did not provide feedback for them.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        <h3>Web Development</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online."</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
+      <div class="services-box">
+        <i class='bx bx-bar-chart-alt-2'></i>
+        <h3>Digital Marketing</h3>
+        <p>We offer comprehensive services in web development, graphic design, and digital marketing. As junior web developers, we create dynamic, user-friendly websites, design captivating graphics, and implement effective digital marketing strategies to help your business thrive online."</p>
+        <a href="#" class="btn">Read More</a>
+      </div>
```
Comment (ahzamir) [inline index.html]: Screenshot 2024-06-06 124419

- Try to add the actual and real data about your services here and also if there is any link for your services add the link (a) tag other than that you can remove them.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        </div>
+      </div>
+      <div class="portfolio-box">
+        <img src="img/Screenshot (5).png" alt="">
+        <div class="portfolio-layer">
+          <h4>Web Design</h4>
+          <p>Coffee Shop web Page that help people order and reserve the place for thier self</p>
+          <a href="#"><i class='bx bx-link-external'></i></a>
+        </div>
+      </div>
```
Comment (ahzamir) [inline index.html]: Screenshot 2024-06-06 124436

- Try to add the Source Code (Your project github link) besides your Demo (Live link).

File: style.css
Comment (ahzamir) [inline style.css:1]: Screenshot 2024-06-06 124343

- Although your design is impressive, your webpage is not responsive and does not look great on smaller screens like the mobile version.
  - Try to make your webpage responsive using media query.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Saqibi4213 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- You have done a great job creating an amazing designed portfolio but still, your portfolio misses some of the Important requirements to be shared.
  - Check the comments below for more details.
  
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
❌Your webpage is responsive for different screen sizes

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by fatiabdulahi** [comment]
> This is Portfolio about ME ! i worked on it with patient and alot of eforts trying to make portfolio professional and i worked on it with two code languages HTML , CSS linters also added and deployed link added live link also mentioned my partner to review my repo and project

**Comment by fatiabdulahi** [comment]
@Saqibi4213 hello dear @Saqibi4213 your work is very well excellent

File: style.css
Comment (Saqibi4213) [reply style.css:1]: hello sir can you please give me brief advice and where is the problem i cant undrenstand i used to try mobile version to make my mobile version responsive too but in screenshot where is the exact problem i cant undrestand

**Comment by Saqibi4213** [comment]
Hello Sir @ahzamir please Review my repo and project 
source link added 
cv download added 
responsive try to make it better 
please if thier is anything left or wrong again give me feedback  thank you

**Comment by Saqibi4213** [comment]
Hello Sir @ahzamir please Review my repo and project
source link added
cv download added
responsive try to make it better
please if their is anything left or wrong again give me feedback thank you

**Review by ahzamir — APPROVED** [review]
### Hi @Saqibi4213 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Capstone project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ahzamir** [comment]
@Saqibi4213 

- Deploy your project after you merged this pull-request
- Well done mehria, don't forget to share your success on the FB group and your LinkedIn profile as this is mandatory.

File: style.css
Comment (fatiabdulahi) [reply style.css:1]: you are work good  dear excellent @Saqibi4213

**Pull request merged**